src/databricks/labs/lakebridge/config.py

```diff
@@ -185,6 +185,6 @@ class ReconcileConfig:
 
 
 @dataclass
-class RemorphConfigs:
+class LakebridgeConfigs:
```

Comment:
I'm fine with this, although to be honest the name doesn't tell me anything about what it is.

For some reason the plural form ("Config**s**") grates a little, and I've been around Java too long because `LakebridgeConfiguration` would feel more natural. :-/